File: ExamSystemAPI/ExamSystemAPI/Models/Entities/Exam.cs

public class Exam
{
    public int Id { get; set; }
    public string Title { get; set; } = null!;
    public string Description { get; set; } = null!;
    public int CreatedByAdminId { get; set; }
    public Admin CreatedByAdmin { get; set; } = null!;

    public ICollection<Question> Questions { get; set; } = new List<Question>();
}

File: ExamSystem-Angular/src/app/core/interface/exam.ts
export interface Question {
  id: number;
  text: string;
  choiceA: string;
  choiceB: string;
  choiceC: string;
  choiceD: string;
  correctAnswer: string;
}

export interface Exam {
  id: number;
  title: string;
  description: string;
  createdBy: number;
  createdByName: string;
  questions: Question[];
}
